modeling/model_configuration/classes.attributes.json

```diff
@@ -149,6 +160,35 @@
     "id": "3T-SBOM-EMS-Artifact-Document-signature",
     "reverseName": "document"
   },
+  {
+    "class": "ExternalDocumentRef",
+    "name": "identifier",
+    "description": "The identifier of the document, unique within the namespace.",
+    "type": "http://www.omg.org/spec/UML/20131001/PrimitiveTypes.xmi#String",
+    "multiplicity": "1",
+    "package": "Artifact",
+    "id": "3T-SBOM-EMS-Artifact-ExternalDocumentRef-identifier"
+  },
+  {
+    "class": "ExternalDocumentRef",
+    "name": "namespace",
+    "description": "The namespace of the identifier of the document.",
+    "type": "http://www.omg.org/spec/UML/20131001/PrimitiveTypes.xmi#String",
+    "multiplicity": "1",
+    "package": "Artifact",
+    "id": "3T-SBOM-EMS-Artifact-ExternalDocumentRef-namespace"
+  },
+  {
+    "class": "ExternalDocumentRef",
+    "name": "signature",
+    "description": "The elelement capturing the optional signature of the document.",
+    "type": "3T-SBOM-EMS-Artifact-Signature",
+    "multiplicity": "0..1",
+    "aggregation": "composite",
+    "package": "Artifact",
+    "id": "3T-SBOM-EMS-Artifact-ExternalDocumentRef-signature",
+    "reverseName": "document"
+  },
   {
     "class": "AbstractArtifact",

```

Comment:
propose adding one more field checksum with a type 3T-...Checksum and a multiplicity of 0..1.  2 reasons to add - checksums are much easier to generate than signatures and it would be compatible with SPDX.